README.md

```diff
@@ -13,69 +13,47 @@ description: "Code which demonstrates how to set up and operationalize an MLOps
 
 [![Build Status](https://aidemos.visualstudio.com/MLOps/_apis/build/status/microsoft.MLOpsPython?branchName=master)](https://aidemos.visualstudio.com/MLOps/_build/latest?definitionId=151&branchName=master)
 
-MLOps will help you to understand how to build the Continuous Integration and Continuous Delivery pipeline for a ML/AI project. We will be using the Azure DevOps Project for build and release/deployment pipelines along with Azure ML services for model retraining pipeline, model management and operationalization.
+MLOps will help you to understand how to build a Continuous Integration and Continuous Delivery pipeline for an ML/AI project. We will be using the Azure DevOps Project for build and release/deployment pipelines along with Azure ML services for model retraining pipeline, model management and operationalization.
 
 ![ML lifecycle](/docs/images/ml-lifecycle.png)
 
-This template contains code and pipeline definition for a machine learning project demonstrating how to automate an end to end ML/AI workflow. The build pipelines include DevOps tasks for data sanity test, unit test, model training on different compute targets, model version management, model evaluation/model selection, model deployment as realtime web service, staged deployment to QA/prod and integration testing.
+This template contains code and pipeline definitions for a machine learning project that demonstrates how to automate an end to end ML/AI workflow.
 
-## Prerequisite
+## Architecture and Features
 
-- Active Azure subscription
-- At least contributor access to Azure subscription
+Architecture Reference: [Machine learning operationalization (MLOps) for Python models using Azure Machine Learning](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/ai/mlops-python)
 
-## Getting Started
-
-To deploy this solution in your subscription, follow the manual instructions in the [getting started](docs/getting_started.md) doc
-
-## Architecture Diagram
-
-This reference architecture shows how to implement continuous integration (CI), continuous delivery (CD), and retraining pipeline for an AI application using Azure DevOps and Azure Machine Learning. The solution is built on the scikit-learn diabetes dataset but can be easily adapted for any AI scenario and other popular build systems such as Jenkins and Travis.
+This reference architecture shows how to implement continuous integration (CI), continuous delivery (CD), and retraining pipeline for an AI application using Azure DevOps and [Azure Machine Learning](/azure/machine-learning/service/overview-what-is-azure-ml). The solution is built on the scikit-learn diabetes dataset but can be easily adapted for any AI scenario and other popular build systems such as Jenkins and Travis.
 
 ![Architecture](/docs/images/main-flow.png)
```

Comment:
Should we just get rid of the .png here and leave it to the architecture page.  Thoughts @eedorenko?

Reply:
Sure, we don't need the diagram here.